lib/Parse/ParseDecl.cpp

```diff
@@ -4349,6 +4349,12 @@ ParserStatus Parser::parseDeclAttribute(DeclAttributes &Attributes,
   checkInvalidAttrName("_functionBuilder", "resultBuilder",
                        DeclAttrKind::ResultBuilder, diag::attr_renamed_warning);
 
+  // Historical name for @section and @used
+  checkInvalidAttrName("_section", "section", DeclAttrKind::Section,
+                       diag::attr_name_close_match);
+  checkInvalidAttrName("_used", "used", DeclAttrKind::Used,
+                       diag::attr_name_close_match);
```

Comment:
These being errors is going to break code that enabled the experimental version of this feature. Can we use `diag::attr_renamed_warning` instead?